prac_08/taxi_simulator.py

```diff
@@ -0,0 +1,84 @@
+"""
+CP1404/CP5632 Practical
+Taxi Simulator
+"""
+from prac_08.taxi import Taxi
+from prac_08.silver_service_taxi import SilverServiceTaxi
+
+MENU = """q)uit, c)hoose taxi, d)rive"""
+
+
+def main():
+    """Code for taxi simulator."""
+    current_taxi = None
+    print(MENU)
+    taxis = [Taxi("Prius", 100), SilverServiceTaxi("Limo", 100, 2), SilverServiceTaxi("Hummer", 200, 4)]
+    current_bill = 0.00
+    choice = input(">>> ").upper()
+
+    while choice != 'Q':
+
+        if choice == 'D':
+            if current_taxi != None:
+                taxis[current_taxi].start_fare()
+                distance = get_distance()
+                taxis[current_taxi].drive(distance)
```

Comment:
`taxis[current_taxi]` is misleading. I should be able to drive my current_taxi, not just use it to get an actual taxi.